tests/functional/lib/helpers.js

```diff
@@ -176,19 +176,24 @@ define([
    *
    * @param {String} selector
    *        QSA compatible selector string
+   * @param {Object} options
+   *        options include polling `timeout`, element `offsetWidth` and element `offsetHeight`
    */
-  function visibleByQSA(selector, timeout) {
-    timeout = timeout || 10000;
+  function visibleByQSA(selector, options) {
```

Comment:
You can create another helper just for ERROR notifications that are always 10px height so we don't have to repeat ourselves with `offsetHeight` https://github.com/mozilla/fxa-content-server/pull/3535#issuecomment-188049066